app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -424,6 +432,20 @@ class HomePresenter(
         }
     }
 
+    override fun onSellClicked() {
+        sellAnalytics.logSellSubmitClicked()
```

Comment:
you can inject analytics inside Fragment to remove jumping from presenter to view just for this

Reply:
I think it's more clear, yeah for now it's dump, but maybe we will need to add some logic in future xD
it can not be avoided ))